backend/api/Controllers/MissionController.cs

```diff
@@ -141,20 +216,114 @@ public async Task<ActionResult<byte[]>> GetMap([FromRoute] string assetCode, str
     }
 
     /// <summary>
-    /// Schedule a new mission
+    /// Reschedule an existing mission
     /// </summary>
     /// <remarks>
-    /// <para> This query schedules a new mission and adds it to the database </para>
+    /// <para> This query reschedules an existing mission and adds it to the database </para>
+    /// </remarks>
+    [HttpPost("{missionId}/reschedule")]
+    [Authorize(Roles = Role.User)]
+    [ProducesResponseType(typeof(MissionRun), StatusCodes.Status201Created)]
+    [ProducesResponseType(StatusCodes.Status400BadRequest)]
+    [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+    [ProducesResponseType(StatusCodes.Status403Forbidden)]
+    [ProducesResponseType(StatusCodes.Status404NotFound)]
+    [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+    public async Task<ActionResult<MissionRun>> Rechedule(
+        [FromRoute] string missionId,
+        [FromBody] RescheduleMissionQuery rescheduledMissionQuery
+    )
```

Comment:
Would include the missionId in the query object and thus in the body, not as part of the query string.